.coveragerc

```diff
@@ -1,5 +1,9 @@
 [report]
 # Regexes for lines to exclude from consideration
 exclude_lines =
+    # Re-enable the standard pragma
+    pragma: no cover
```

Comment:
What does this do? What is "the standard pragma", and why does it need to be re-enabled?